spec/support/shared_examples_for_object_equality.rb

```diff
@@ -0,0 +1,59 @@
+
+# Examples for making sure two objects are equal
```

Comment:
Would it be possible put these examples in the `TestUtils` file? I think that may be a better place for them, as that's where they live in the original codebase.